.github/workflows/java.yml

```diff
@@ -23,7 +23,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ ubuntu-latest ]
-        java: [ 8, 9, 10, 11, 12, 13, 14, 15, 16, 17 ]
+        java-version: [ 8, 11, 17, 20, 21 ]
```

Comment:
I'm removed some old setups and added new ones. This mimics the setup we have for truelayer-java